n3fit/src/evolven3fit_new/evolve.py

```diff
@@ -90,20 +87,21 @@ def evolve_fit(
     if eko_path is not None:
         eko_path = pathlib.Path(eko_path)
         _logger.info(f"Loading eko from : {eko_path}")
-        eko_op = output.Output.load_tar(eko_path)
+        eko_op = eko.EKO.edit(eko_path)
```

Comment:
The suggested way to use this method is within a context manager:

```py
with eko.EKO.edit(eko_path) as eko_op:
    ...
```

But if you are not using the context manager, you are responsible to call `eko_op.close()` by yourself (otherwise terrible things may happen :P)

Reply:
Consider that you can also do it in two steps:
```py
eko_op = eko.EKO.edit(eko_path)

with eko_op:
    ...
```

Reply:
@andreab1997 you resolved the conversation before applying any solution: `eko_op` is not used in a context manager, nor explicitly closed.

Why do you believe it is not required?